moshi/src/test/java/com/squareup/moshi/JsonValueReaderTest.java

```diff
@@ -423,6 +424,18 @@
     }
   }
 
+  @Test public void numberToStringCoersion() throws Exception {
+    JsonReader reader = new JsonValueReader(Arrays.asList(0, 9223372036854775807L, 2.5d, 3.010f, "a", "5"));
```

Comment:
is there any reason to have the trailing `0` in `3.010f` ?

Reply:
nope, it's copy-paste from the other test case. will remove.